code/mob/living/carbon/human.dm

```diff
@@ -116,6 +116,7 @@
 
 	//dismemberment stuff
 	var/datum/human_limbs/limbs = null
+	var/obj/item/organ/head/head_tracker = null //Tracks our head in case we are a skeleton
```

Comment:
See comment about using `head` on skeleton mutantrace (but just renaming it to `head_tracker`).